packages/workorder-angular/package.json

```diff
@@ -17,17 +17,16 @@
     "angular-material": "1.0.7",
     "angular-messages": "1.5.3",
     "angular-ui-router": "0.2.18",
-    "debug": "^2.6.6",
-    "fh-wfm-mediator": "0.3.3",
-    "lodash": "4.7.0",
+    "bluebird": "^3.5.0",
     "q": "1.4.1",
```

Comment:
We might use the $q service from angular, but we can at least remove it from the dependencies